### ISSUE TITLE
[glpi 9.5.2] queued notifications

### ISSUE THREAD

**Issue by cpasDav** [glpi-project/glpi#8293]
Hi,

Since I have upgraded to glpi 9.5.2 (from 9.4.7), only email notification for New Ticket are sent correctly, all the others stuck in the notification queue.

Email notification is set to CLI.
Automatic action for queue notification is running.
Notification for events like ticket follow up/ticket resolved/ticket closed are activated and both are supposed to be sent to same recipients as for New Ticket event.

1. create a test ticket
2. Add a follow up message or mark the ticket as resolved or close the ticket
3. check the notification queue
4. notifications are stuck there

**Expected behavior**

All notifications are supposed to be sent out of the queue.

**Your GLPI setup (you can find it in Setup > General menu, System tab)**

```
GLPI 9.5.2 (/glpi => /var/www/html/glpi)
Installation mode: TARBALL
Server: CentOs
DB: MariaDb
```

**Comment by cpasDav** [comment]
When I go to the notification queue, if I check all queued items and select the action "send", all queued items are sent correctly.

**Comment by TossMySalad** [comment]
I have the same issue.

**Comment by cpasDav** [comment]
@TossMySalad, there is clearly a problem, fyi I could unblock the situation like this:
- delete all messages from the notification queue
- Reset to zero the last execution of queuednotification from automatic action

**Comment by StbanMc** [comment]
I present the same problem, it only sends the new records, the rest gets stuck.

I think it is because my regional configuration is different from the server versus mysql, but because it is a shared hosting I cannot modify it and they are left with future dates.

**Comment by cedric-anne** [comment]
Hi,

Please consider upgrading to GLPI 9.5.5 (released yesterday).

In #8878, we changed the way crontask errors are handled. Now, when a task fails, the error should be logged correctly in `php-errors.log`, and task status should be put back to "waiting" state. If a task fails too many times (5 fails on last 10 executions), a notification will be sent (notification is configurable using `Automatic actions` itemtype and `Monitoring of automatic actions` event).

With this, you should be able to provide us an error stack trace, which can help us to investigate.

Regards

**Comment by trasher** [comment]
Should be fixed, https://github.com/glpi-project/glpi/issues/8836#issuecomment-821137384``
